test/karma.conf.js

```diff
@@ -9,6 +9,13 @@ const browserStack = {
 
 process.env.CHROME_BIN = playwright.chromium.executablePath();
 
+// BrowserStack rate limit after 1600 calls every 5 minutes.
+// Per second, https://www.browserstack.com/docs/automate/api-reference/selenium/introduction#rest-api-projects
+const MAX_REQUEST_BROWSERSTACK = 1600 / (60 * 5);
```

Comment:
```suggestion
const MAX_REQUESTS_PER_SECOND_BROWSERSTACK = 1600 / (60 * 5);
```
The maximum **number** of requests in browserstack would be just 1600.